### PR TITLE
Introducing a "maximumTilt" property for the Camera

### PR DESCRIPTION
**Pull request by fnicollet**
Introduced a maximumTilt property for the Camera, that, if present, limits the camera's tilt.

Here is a sandbox sample to showcase this behaviour:
```
var viewer = new Cesium.Viewer("cesiumContainer");
var camera = viewer.camera;
camera.maximumTilt = 1.2;

camera.setView({
  destination: new Cesium.Cartesian3(
    1216394.1392207467,
    -4736348.59346919,
    4081293.9160685353
  ),
  orientation: {
    heading: 0.018509338875732695,
    pitch: -0.09272999615872646,
  },
});
```
The camera's pitch is limited (wether you use mousewheel-click + up/down or Ctrl+mouse up/down), which is currently impossible to do with the Cesium public API unless you re-code all the mouse/keyboard/touch gestures.

The maximumTilt value is expressed in radians. Degrees might have been more dev-friendly but most things seem to be expressed in radians in Cesium.

Follows-up on this PR which was never completed:
https://github.com/CesiumGS/cesium/pull/7428

Added Business Geografic/fnicollet to the Contributors
The CLA for Business Geografic has been signed today.

Added a line in the CHANGES.md file. Should I wait for the PR to be created and update the file with the PR link or someone from Cesium will?

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @fnicollet!
* :grey_exclamation: There was an error checking the CLA! If this is your first contribution, please send in a [Contributor License Agreement](https://github.com/CesiumGS/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla).
    * Maintainers, this was the error I ran into while attempting to process the CLA check. Please resolve it to continue CLA checking.
    ```
    Error: The service is currently unavailable.
    ```
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

**Comment by OmarShehata** [comment]
I can confirm we've received your corporate CLA.

**Comment by fnicollet** [comment]
Hello, 
Is there any change necessary before this PR can be reviewed?

Thanks,
Fabien

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @fnicollet! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @fnicollet! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by jailln** [comment]
Hello,
Is there any news on this PR ? 
We would also really benefit from this feature in our project.
Thanks!

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @fnicollet! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by fnicollet** [comment]
Hello @OmarShehata ,

could this PR get reviewed please ? 

Thanks,
Fabien

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @fnicollet! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @fnicollet! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @fnicollet! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

**Comment by ebogo1** [comment]
Might fix https://github.com/CesiumGS/cesium/issues/9689?

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @fnicollet! 

No one has commented on this pull request in 90 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 90 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

File: Source/Scene/Camera.js [hunk 1]
```diff
@@ -177,6 +177,12 @@ function Camera(scene) {
    * @default 100000.0;
    */
   this.defaultZoomAmount = 100000.0;
+  /**
+   * If set, the camera will not be able to tilt past this angle, expressed in radians.
+   * @type {Number}
+   * @default undefined
+   */
+  this.maximumTilt = undefined;
```
Comment (sanjeetsuhag) [inline Source/Scene/Camera.js:185]: Similar to `maximumZoomAmount` and `minimumZoomAmount`, any interaction related limitations are better suited for the `ScreenSpaceCameraController`.

**Review by sanjeetsuhag — COMMENTED** [review]
Thanks for your contribution @fnicollet. From the discussions, this seems like a feature a lot of users might like. I have a small suggestion regarding the public API for this. Additionally, we expect all new features to have good test coverage, so it would be helpful if you could add those too.

**Comment by fnicollet** [comment]
Hello and thanks for your message,

This PR was added in September 2020 and since then, we are in the process or switching to another 3D library, so I won't really have time to work on those changes you ask for, especially since I have no idea how to write tests for Cesium for this specific use case.
Hopefully someone will take over and it gets added to Cesium, the code is quite straightforward, or maybe someone from the Cesium team can implement it

Fabien

**Pull request closed without merging**